src/Bpi/ApiBundle/Tests/WritingNodeTest.php

```diff
@@ -93,8 +95,19 @@ public function testPush()
             'type' => $this->faker->name,
             'category' => 'Litteratur',
             'audience' => 'Voksne',
-            'tags' => implode(',', $tags)
-            // TODO: Add images payload.
+            'tags' => implode(',', $tags),
+            'assets' => [
+                [
+                    'path' => 'https://picsum.photos/200',
```

Comment:
Consider using a local image file to prevent dependency on external service.

Reply:
This was the initial idea, yet thought to skip having controllers returning image that would be used only for tests.